flatten_dict/flatten_dict.py

```diff
@@ -99,6 +102,7 @@ def unflatten(d, splitter='tuple', inverse=False):
         used to split.
         'tuple': Use each element in the tuple key as the key of the unflattened dict.
         'path': Use `pathlib.Path.parts` to split keys.
+        'underscore': Use underscores to join keys.
```

Comment:
Don't forget the type above: `{'tuple', 'path', Callable}` -> `{'tuple', 'path', 'underscore', Callable}`